### ISSUE TITLE
build task cacophony / debounce audio cues

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#164921]
When running a build task (like VS Code's build task) w the new fail/success audio cues, it sounds like the failure noise starts to play and then the success one is overlaid. We should make sure to manage this better

**Comment by meganrogge** [comment]
<img width="979" alt="Screen Shot 2022-10-31 at 1 34 23 PM" src="https://user-images.githubusercontent.com/29464607/199072370-f2300773-04ac-4dd4-a352-f9b6a4c00626.png">

the audio cue is always enabled here

https://github.com/microsoft/vscode/blob/287d963f467d4c309e2e3fef804b49d567c92b93/src/vs/workbench/contrib/audioCues/browser/audioCueService.ts#L41-L48

**Comment by meganrogge** [comment]
@hediet are consumers of this expected to handle repeat requests?

**Comment by hediet** [comment]
Currently, the audio cue service does not do any debouncing.

What do you suggest?

**Comment by meganrogge** [comment]
I would suggest the audio cue service drops any requests to play a sound that's already playing

**Comment by hediet** [comment]
Sounds reasonable.

**Comment by meganrogge** [comment]
I'd suggest the same thing for `playSound` which would fix https://github.com/microsoft/vscode/issues/163731
